new(mc): added HPE Primera MC doc (#3457)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/hardware-storage-hpe-primera-restapi.md

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                                   | Service Description    | Discovery  |
|:--------------|:---------------------------------------------------|:-----------------------|:----------:|
| Disk-Usage    | HW-Storage-HPE-Primera-Disk-Usage-RESTAPI-custom   | Check the disk usage   | X          |
| Volume-Usage  | HW-Storage-HPE-Primera-Volume-Usage-RESTAPI-custom | Check the volume usage | X          |

> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.

> If **Discovery** is checked, it means a service discovery rule exists for this service template.

</TabItem>
</Tabs>

### Discovery rules

#### Service discovery

| Rule name                                           | Description                                                |
|:----------------------------------------------------|:-----------------------------------------------------------|

> | HW-Storage-HPE-Primera-RESTAPI-Disk-Status-Id       | Découvre les disques physiques et supervise le statut.     |

Reviewer: EN dans FR pour tout le tableau

> | HW-Storage-HPE-Primera-RESTAPI-Disk-Status-Id       | Découvre les disques physiques et supervise le statut.     |
> | HW-Storage-HPE-Primera-RESTAPI-Disk-Status-Position | Découvre les disques physiques et supervise le statut.     |
> | HW-Storage-HPE-Primera-RESTAPI-Disk-Usage-Id        | Découvre les disques physiques et supervise l'utilisation. |
> | HW-Storage-HPE-Primera-RESTAPI-Disk-Usage-Position  | Découvre les disques physiques et supervise l'utilisation. |
> | HW-Storage-HPE-Primera-RESTAPI-Volume-Usage-Id      | Découvre les volumes et supervise l'utilisation.           |
> | HW-Storage-HPE-Primera-RESTAPI-Volume-Usage-Name    | Découvre les volumes et supervise l'utilisation.           |

Author: ```suggestion
| HW-Storage-HPE-Primera-RESTAPI-Disk-Status-Id       | Discover the physical disks and monitor their state.     |
| HW-Storage-HPE-Primera-RESTAPI-Disk-Status-Position | Discover the physical disks and monitor their state.     |
| HW-Storage-HPE-Primera-RESTAPI-Disk-Usage-Id        | Discover the physical disks and monitor their usage. |
| HW-Storage-HPE-Primera-RESTAPI-Disk-Usage-Position  | Discover the physical disks and monitor their usage. |
| HW-Storage-HPE-Primera-RESTAPI-Volume-Usage-Id      | Discover the volumes and monitor their usage.           |
| HW-Storage-HPE-Primera-RESTAPI-Volume-Usage-Name    | Discover the volumes and monitor their usage.           |
```
